projects/azure/cluster-wide.yaml

```diff
@@ -24,3 +24,4 @@ items:
   - pweil-
   - thekad
   - y-cote
+  - gburges
```

Comment:
Can you keep this sorted like the rest of the entries?

Reply:
done